pom.xml

```diff
@@ -47,8 +47,8 @@
     <scm>
         <connection>scm:git:git://github.com/jenkinsci/envinject-plugin.git</connection>
         <developerConnection>scm:git:git@github.com:jenkinsci/envinject-plugin.git</developerConnection>
-      <tag>HEAD</tag>
-  </scm>
+        <tag>HEAD</tag>
```

Comment:
@oleg-nenashev IMO, are related. I'm reviewing deprecated things and incorrect details. If you prefer, I can add it to the description.